pymongo/auth_shared.py

```diff
@@ -78,7 +78,7 @@ def __hash__(self) -> int:
 
 
 GSSAPIProperties = namedtuple(
-    "GSSAPIProperties", ["service_name", "canonicalize_host_name", "service_realm"]
+    "GSSAPIProperties", ["service_name", "canonicalize_host_name", "service_realm", "service_host"]
```

Comment:
It doesn't look like we actually do anything with service_host. Why add it if it's not used?